src/main/java/com/minecolonies/coremod/util/constants/RenderUtils.java

```diff
@@ -0,0 +1,137 @@
+package com.minecolonies.coremod.util.constants;
+
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.colony.ColonyView;
+import com.minecolonies.coremod.configuration.Configurations;
+import com.minecolonies.coremod.util.BlockPosUtil;
+import com.minecolonies.structures.helpers.Settings;
+import com.minecolonies.structures.helpers.Structure;
+import com.minecolonies.structures.lib.ModelHolder;
+import net.minecraft.block.Block;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.entity.EntityPlayerSP;
+import net.minecraft.client.multiplayer.WorldClient;
+import net.minecraft.client.renderer.block.model.IBakedModel;
+import net.minecraft.entity.item.EntityEnderCrystal;
+import net.minecraft.init.Blocks;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.math.BlockPos;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import static sun.awt.geom.Curve.next;
+
+/**
+ * Used for some rendering purpose like waypoint rendering.
+ */
+public final class RenderUtils
+{
+    /**
+     * Half point offset to get in the middle of a block.
+     */
+    private static final double HALF_BLOCK_OFFSET = 0.5;
+
+    /**
+     * Offset where the waypoint rendering marks to.
+     */
+    private static final int FIX_POINT_OFFSET = 10;
+
+    /**
+     * List of all BlockPos in the colony border.
+     */
+    public static final List<BlockPos> colonyBorder = new ArrayList<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this member "protected". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2386)